README.md

```diff
@@ -9,6 +9,29 @@ World of Warcraft addon for the Hardcore community
 - (Hardcore Rules) Prevents Mailbox and Auction House Access
 - (Bubble Hearth) Warns the player while Bubble hearthing. Also reports to guild
 
+## FAQ
+### Addon says active time is much lower than played time
+
+The addon tracks a certain percentage of your addon uptime compared to total /played. If it is about seconds or minutes - no problem. If it is about hours - consider rerolling before level 20. After level 20, record the run as instructed by the error message.
+
+this can happen due to: 
+- Disconnect or Game crash. Character remains in the online server while the locally stored addon is severed from tracking it. 
```

Comment:
The same as you wrote under "The addon is missing recorded levels" too. The saved variables get never updated when the game crashes. Could add it here too, as this is the most likely cause of data loss.